tutorials/tutorial_structure/index.md

```diff
@@ -0,0 +1,296 @@
+
+You will note a box called `Data files and scripts` in the upper left-hand corner of this webpage.
+Please download the files listed in this directory.
+Drag and drop `example_file.nex` into your `data` directory.
+Then, move `test.Rev` into your `scripts` directory.
+Having a directory of data that contains all your data for a project, and a directory of scripts is a good practice. 
+This allows you to stay organized and avoid misplacing crucial scripts in your data analysis pipeline.
+
+
+{% aside Spaces In Filenames %}
+Most scientific programming languages and software does not deal well with spaces in file names.
+If you will be doing much scientific computing, it will be best to get in the habit of not using spaces in file and folder names.
+{% endaside %}
+
+
+{% subsection Preparing to Run RevBayes %}
+
+In this section of the tutorial, we will focus on running RevBayes from your computer's graphical interface. 
+First, note down the location of your tutorial directory. 
+For example, mine is in my computer's user home, in a directory called `Tutorials`.
+A graphic of this is shown in Fig. 1.
+
+
+Computers do not understand the visual information of the directory structure.
+Instead, we will translate this information into text.
+
+{% figure example %}
+<img src="figures/Directory.png" width="600">
+{% figcaption %}
+This is an example of the Macintosh File Viewer. In this instance, I have adirectory, `Tutorials` with a subdirectory for this specific tutorial.
+{% endfigcaption %}
+{% endfigure %}
+  
+
+{% subsubsection Macintosh %}
+
+The home directory on a computer on a Mac is labeled with a `~/`. 
+Each contained directory is separated by a `/` character.
+The above directory structure would be written out like so:
+
+```
+~/tutorials/tutorial_structure/
+```
+{:.bash}
+
+Note down where you have your tutorial stored.
+
+Next, we will launch RevBayes. 
+Do this by double-clicking on `rb`.
+We will now set your working directory. 
+This ensures that RevBayes is aware of where in your computer your code and data are stored.
+By default, RevBayes assumes you are in your user home on your computer.
+Therefore, we can leave off the `~/`, as RevBayes will assume its presence. 
+In RevBayes, use the `setwd()` command in conjunction with your path to your tutorial to set your working directory. 
+For example, my command will look like so:
+
+```
+setwd("Tutorials/tutorial_structure/")
+```
+{:.bash}
+
+Finally, test your working directory like so:
+
+```
+ source("scripts/test.Rev")
+```
+
+If everything has suceeded, you will see the following output:
+
+
+```
+   Processing file "scripts/test.Rev"
+   Hi there! Welcome to RevBayes! I am now going to read in some test data.
+   Successfully read one character matrix from file 'data/primates_and_galeopterus_cytb.nex'
+   Congratulations, you set up your scripts and code directories correctly.
+```
+{:.Rev-output}
+
```

Comment:
{% aside If something goes wrong %}
If you were not able to successfully execute the script, the most common culprit is that RevBayes is not executing from where you think. Try running `getwd()` and making sure that your starting working directory is what you think it is.
{% endaside %}